test/logger-instantiation.js

```diff
@@ -202,7 +215,9 @@ test('Logger instance properties', async (t) => {
     , mac: '01:02:03:04:05:06'
     , tags: ['whiz', null, undefined, '', ' ', '\t', '\n', 'bang', 'done', 1234, 0]
     , ignoreRetryableErrors: false
+    , verboseEvents: true
     , sendUserAgent: false
+    , maxAttempts: 5
```

Comment:
Thanks for adding this. One of these days maybe we'll put in an expected assertion count or something so that we know that they're all covered. That would still be error prone if we don't bump the count, though, but at least that would be developer error.

Reply:
I also wondered if we could write a test that would parse the README and verify that every option was documented,